wip/jstachio-1.0.0.buildspec

```diff
@@ -0,0 +1,21 @@
+groupId=io.jstach
+artifactId=jstachio
+version=1.0.0
+
+display=${groupId}:${artifactId}
+
+gitRepo=https://github.com/jstachio/${artifactId}.git
+gitTag=v${version}
+
+tool=mvn-3.9.2
+jdk=17
+newline=lf
```

Comment:
need to add `umask=022`
my own laptop has 002 default umask, and this makes a difference in result